Runtime/WebEventManager.cs

```diff
@@ -98,6 +100,15 @@ private static void parseEventPayload(string strPayload)
                 CardValid = validation.data.data.valid;
                 CardValidationChanged?.Invoke(validation);
                 break;
+            case "fingerprint":
+                var fingerprint = StringEvent<FingerprintPayload>.FromJSON(strPayload);
+                 if (fingerprint == null) 
+                 { 
+                    break;
```

Comment:
our API accepts fingerprint empty values, so we allow the user to transact with an unknown device
```suggestion
                FingerprintVisitorId = ""
                FingerprintRequestId = ""
```

Reply:
These values are already empty strings if we don’t set them here